main.go

```diff
@@ -145,10 +149,15 @@ func mainerr() error {
 			}
 			gopath = filepath.Join(uhd, "go")
 		}
-
 		// TODO I don't think the module cache path is advertised anywhere public...
 		// intentionally but in case it is, replace what follows
-		localCacheProxy = "GOPROXY=file://" + path.Join(filepath.ToSlash(gopath), "pkg", "mod", "cache", "download")
+		cachePath := path.Join(filepath.ToSlash(gopath), "pkg", "mod", "cache", "download")
+		if goenv.ReleaseTags["go1.13"] && cachePath[0] != '/' {
```

Comment:
Sadly it seems there was a bug pre 1.13 in the handling of the `GOPROXY` value. i.e. if we add the third slash things break pre 1.13, and if we don't then 1.13 breaks.